clio-client/src/test/scala/org/broadinstitute/clio/client/webclient/ClioWebClientSpec.scala

```diff
@@ -28,6 +28,7 @@ class ClioWebClientSpec extends BaseClientSpec with AsyncMockFactory {
 
   //private val index = ModelMockIndex()
   private val timeout = 1.second
+  private val idleTimeout = 1.second
```

Comment:
Can you add a test that verifies that the new idle timeout setting is having an effect? Or is there an existing test that already asserts this?

Reply:
Can't comment on it, but it's already tested on line 131. Instead of using the `requestTimeout` to set the idle timeout like we were before, this uses an explicit `idleTimout` to do so.